### DISCUSSION TITLE
Stumped on #15: getting 96.9% match but both images looks identical.

### DISCUSSION THREAD

**Discussion by damerval** [cssbattle/forum#76] [Bugs reports]
I have gone every which way I knew to find out what gave, but I am officially stumped and any help would be greatly appreciated. 
![Screenshot 2021-06-30 105953](https://user-images.githubusercontent.com/16507160/124017837-a9601b80-d993-11eb-9abb-1e1f1d36a0b0.png)

My markup (it's not code, people :))

```
<div id="midbox">
  <div id="c3"></div>
  <div id="c1">
    <div id="c2"></div>
  </div>
</div>
<style>
  * {
    position: relative;
    background-color: #09042A;
  }
  #midbox {
    width: 250px;
    height: 150px;
    margin: auto;
    margin-top: 75px  
  }
  div[id*=c] {
    background-color: #7B3F61;
    position: absolute; 
    top: 0; bottom: 0;
  }
  #c1 {
    height: 150px; width: 150px;
    border-radius: 50%;
    left: 0;
    overflow: hidden;
  }
  #c2 {
    background-color: #000;
    left: 100px;
    height: 150px; width: 150px;
    border-radius: 50%;  
  }  
  #c3 {
    background-color: #E78481;
    height: 150px; width: 150px;
    border-radius: 50%;
    right: 0;
  }
</style>

```

**Comment by chinchang** [comment]
Hi @damerval There is difference actually:
![CleanShot 2021-07-01 at 00 39 54](https://user-images.githubusercontent.com/379918/124017932-ec0e0c00-da04-11eb-8484-185647a966af.png)

See the part in center, thats the diff

**Reply by damerval** [reply]
I just saw that, can't believe I was so dumb. Is there a diff tool I can use to avoid embarrassing myself like this in the future? :)

**Reply by chinchang** [reply]
Coming up on the website real soon :)

**Reply by gunnarbittersmann** [reply]
> I just saw that, can't believe I was so dumb. Is there a diff tool I can use to avoid embarrassing myself like this in the future? :)

There is. https://codepen.io/alexzaworski/full/vwmNKr
